examples/sid_visualization/README.md

```diff
@@ -0,0 +1,135 @@
+# SID Visualization Example
+
+## Prerequisites
+
+To run the demo you will need the following:
+- Docker
+- NodeJS
+  - See the installation guide [here](https://nodejs.org/en/download/)
+- `yarn`
+  - Once NodeJS is installed with NPM, run `npm install --global yarn` to install yarn
+- `docker-compose` (Tested with version 1.29)
+
+## Setup
+
+To run this demo, ensure all submodules are checked out
+
+```bash
+git submodule update --init --recursive
+```
+
+### Build Morpheus Dev Container
+
+Before launching the demo, we need the dev container for Morpheus to be created:
+
+```bash
+export DOCKER_IMAGE_TAG="sid-viz"
+
+# Build the dev container
+./docker/build_container_dev.sh
+```
+
+### Launch User Interface
+
+We will use docker-compose to build and run the entire demo. To launch everything, run the following from the repo root:
+
+```bash
+# Save the Morpheus repo directory
+export MORPHEUS_HOME=$(git rev-parse --show-toplevel)
+
+# Ensure SID model is downloaded for deployment to Triton
+./scripts/fetch_data.py fetch models
+
+# Change to the example directory
+cd ${MORPHEUS_HOME}/examples/sid_visualization
+
+# Launch the containers
+DOCKER_BUILDKIT=1 docker-compose up --build -d
+```
+
+The following GUI should be displayed when all containers have completed launching:
+
+<img src="./img/initial_win.png" width=75% height=75%>
+
+### Build Morpheus
+
+Once docker-compose has been launched, exec into the container to build and run Morpheus:
+
+```bash
+# Exec into the morpheus container
+docker-compose exec morpheus bash
+
+# Inside the container, compile morpheus
+BUILD_DIR=build-docker ./scripts/compile.sh
+
+# Install morpheus with an extra dependency
+pip install -e . && pip install websockets
+
+# Verify Morpheus is installed
+morpheus --version
+
+# Ensure the data has been downloaded
+./scripts/fetch_data.py fetch examples
+```
+
+** Note: Keep this shell in the Morpheus Dev container running. It will be used later to start Morpheus.
+
+## Running the Demo
+
+### Running Morpheus
+
+After the GUI has been launched, Morpheus now needs to be started. In the same shell used to build Morpheus (the one running the Morpheus Dev container), run the following:
+
+```bash
+python examples/sid_visualization/run.py \
+  --debug --use_cpp=False --num_threads=1 \
+  --triton_server_url=triton:8001 \
+  --input_file=./examples/data/sid_visualization/group1-benign-2nodes-v2.jsonlines \
+  --input_file=./examples/data/sid_visualization/group2-benign-50nodes.jsonlines \
+  --input_file=./examples/data/sid_visualization/group3-si-50nodes.jsonlines \
+  --input_file=./examples/data/sid_visualization/group4-benign-49nodes.jsonlines
+```
+
+This launch will use all of the available datasets. Each dataset will show up as one batch in the visualization. Here is a description of each dataset:
```

Comment:
Can we put a note here that it may take a little while when first running to initialize the models in Triton since they need to be converted from ONNX to TRT? After the initial run, things should be quick